Source/Forms/Gen5/IDs5.ui

```diff
@@ -426,6 +426,13 @@
         </property>
        </widget>
       </item>
+      <item row="5" column="0">
+       <widget class="QCheckBox" name="checkBoxXOR">
+        <property name="text">
+         <string>Show XOR PID Results</string>
```

Comment:
This will probably be very confusing to users that are new to Gen 5. It would probably be better to give some indication of what they want to rng (stationary/wild vs egg/event/) or whatever the combos are